odxtools/utils.py

```diff
@@ -43,3 +46,21 @@ def short_name_as_id(obj: Any) -> str:
         return f"_{sn}"
 
     return sn
+
+
+def is_short_name(test_val: str) -> bool:
+    """Returns true iff the test_val string is a ODX short name.
+
+    See also: ISO 22901 section 7.1.1
+    """
+    short_name_pattern = re.compile("[a-zA-Z0-9_]+")
```

Comment:
compiling regex is an expensive operation, you may want to move it out of the function so it only gets done once

Reply:
other than this, LGTM